src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts

```diff
@@ -478,7 +480,7 @@ export class SettingsTreeIndicatorsLabel implements IDisposable {
 					...this.defaultHoverOptions,
 					content,
 					linkHandler: (url: string) => {
-						const [scope, language] = decodeURIComponent(url).split(':');
+						const [scope, language] = SettingScopeLink.parse(url)?.split(':');
```

Comment:
The question mark implies the return type might be undefined but there's no null checks after.